### ISSUE TITLE
Company Portal App Install Sticking at 100%

### ISSUE THREAD

**Issue by YANSA23** [MSUDenverSystemsEngineering/SolidWorks#1]
HI, I came across your repo while trying to figure out how to deploy SW 23 through intune as we move in that direction. I'm able to get it installing via Company Portal, but it seems the install status gets stuck at 100% for hours and hours after it finishes actually installing. This is the only software I've had with this behavior, and I was curious if yall have seen that with your deployments or how you're achieving reliable installs?

**Comment by TM-23** [comment]
Hi @YANSA23, installs of SW do seem to take 100 years and they do appear stuck at times. Have you found a way to increase the reliability and speed of deployment?

**Comment by YANSA23** [comment]
@TM-23 I forgot to update this, but I discovered it's not really SW taking hours and hours, it's Intune. SW itself has been found to be reliably downloading and installing through Intune within about 45 minutes to an hour. 

The % progress bar in Intune is very misleading. I turned on the windows notifications for the install and from the first notification of "downloading and installing" to the "has successfully installed" notification is a much more realistic and reliable way to see when the download/install actually finishes.
